client/src/pages/ProfileListings/alert.tsx

```diff
@@ -0,0 +1,5 @@
+import MuiAlert, { AlertProps } from '@material-ui/lab/Alert';
+
+export default function Alert(props: AlertProps) {
```

Comment:
```suggestion
export default function Alert(props: AlertProps): JSX.Element {
```
Specify the return type of component